GH-131278: Support building using "computed gotos" for clang-cl on Windows (#131279)
python/cpython: PCbuild/readme.txt

for more on this topic.

> Optimization flags
> ------------------

Reviewer: ```suggestion
Optimization flags
------------------

```

> You can set optimization flags either via
>  * environment variables, e.g.
>    set WITH_COMPUTED_GOTOS=true
>  * or pass them as parameters to `build.bat`, e.g.
>    build.bat "/p:WITH_COMPUTED_GOTOS=true"
>  * or put them in `msbuild.rsp` in the `PCbuild` directory, one flag per line

Reviewer: ```suggestion
You can set optimization flags either via

 * environment variables, for example:

     set WITH_COMPUTED_GOTOS=true

 * or pass them as parameters to `build.bat`, for example:

     build.bat "/p:WITH_COMPUTED_GOTOS=true"
 
 * or put them in `msbuild.rsp` in the `PCbuild` directory, one flag per line.
```

> Supported flags are:
>  * WITH_COMPUTED_GOTOS: build the interpreter using "computed gotos".
>    Currently only supported by clang-cl.

Reviewer: ```suggestion
Supported flags are:

 * WITH_COMPUTED_GOTOS: build the interpreter using "computed gotos".
   Currently only supported by clang-cl.
```

Optimization flags
------------------

You can set optimization flags either via

>  * environment variables, for example:

Reviewer: I actually missed it, but there is a space before `*`. You may want to re-align the bullet points to remove that space (same with "supported flags" section).

Author: There is a space everywhere else before the `*`, e.g. ` * python` in line 354, I just copied that.
Is this incorrect?

Reviewer: Oh. Well, I think it doesn't really matter. But if we were to one day convert it to RST, we should remove that space. Let's remove it in new text for now (there's no space before in the "Add a new project" section)
